packages/grafana-ui/src/components/uPlot/config/UPlotAxisBuilder.ts

```diff
@@ -168,7 +168,7 @@ export class UPlotAxisBuilder extends PlotConfigBuilder<AxisProps, Axis> {
     } else if (formatValue) {
       config.values = (u: uPlot, splits, axisIdx, tickSpace, tickIncr) => {
         let decimals = guessDecimals(roundDecimals(tickIncr, 6));
-        return splits.map((v) => formatValue!(v, decimals > 0 ? decimals : undefined));
+        return splits.map((v) => (v == null ? null : formatValue!(v, decimals > 0 ? decimals : undefined)));
```

Comment:
```suggestion
        return splits.map((v) => {
            if (v == null) {
                return null;
            } else {
                return formatValue!(v, decimals > 0 ? decimals : undefined);
            }
        });
```

@leeoniya I know you like short versions better, but maybe for the sake of readability 😁